packages/camera/camera/ios/Classes/QueueUtils.h

```diff
@@ -7,17 +7,13 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /// Queue-specific context data to be associated with the capture session queue.
-extern const char *FLTCaptureSessionQueueSpecific;
-
-/// A class that contains dispatch queue related helper functions.
-@interface QueueHelper : NSObject
+extern const char* FLTCaptureSessionQueueSpecific;
```

Comment:
nits: Is this used?

Reply:
Yep. This is used to verify if an operation is on capture session queue.